src/pages/git/git-pull/index.md

```diff
@@ -3,13 +3,74 @@ title: Git Pull
 ---
 ## Git Pull
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/git/git-pull/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+`git pull` is a Git command used to update the local version of a repository from a remote. 
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+`git pull` one of the four commands that prompts network interaction by Git. By default, `git pull` does two things.
```

Comment:
I would just say, "It is one of the four....." and have it follow up behind the last sentence. So like this: 

`git pull` is a Git command used to update the local version of a repository from a remote. It is one of ....

Thoughts?